microsoft/PowerToys
# runner: initial automatic update

## Summary of the Pull Request
- implement autoupdate flow via action runner and "Download Updates Automatically" functionality/option
- introduce "updating" lib and refactor stuff
- do not show duplicate toast notifications


## PR Checklist
* [x] Applies to #1686, #2028 


## Validation Steps Performed
- changed Version to 0.15 and installed update "now" and "at next launch"
- verified that only a single toast notification appears in action center 
![image](https://user-images.githubusercontent.com/1828123/79361364-df131300-7f4d-11ea-9eb7-58ce8b87e38a.png)
